src/shared/json.c

```diff
@@ -2076,6 +2076,21 @@ int json_variant_merge(JsonVariant **v, JsonVariant *m) {
         return 1;
 }
 
+int json_variant_merge_objectb(JsonVariant **v, ...) {
+        _cleanup_(json_variant_unrefp) JsonVariant *w = NULL;
+        va_list ap;
+        int r;
+
+        va_start(ap, v);
+        r = json_buildv(&w, ap);
+        va_end(ap);
+
```

Comment:
Extra newline (or at least it's not present in the `json_variant_set_fieldb()` and `json_variant_append_arrayb()` helpers.